module/CLI/src/Command/ShortUrl/ListShortUrlsCommand.php

```diff
@@ -152,30 +166,30 @@ protected function execute(InputInterface $input, OutputInterface $output): ?int
         return ExitCodes::EXIT_SUCCESS;
     }
 
-    private function renderPage(OutputInterface $output, bool $showTags, ShortUrlsParams $params, bool $all): Paginator
-    {
+    private function renderPage(
+        OutputInterface $output,
+        array $showParams,
+        ShortUrlsParams $params,
+        bool $all,
```

Comment:
This trailing comma does not work in PHP 7.4
```suggestion
        bool $all
```